Formula/aws-sam-cli.rb

```diff
@@ -7,10 +7,10 @@ class AwsSamCli < Formula
   config_provider = ConfigProvider.new('aws-sam-cli')
 
   desc "AWS SAM CLI 🐿 is a tool for local development and testing of Serverless applications"
-  homepage "https://github.com/awslabs/aws-sam-cli/"
+  homepage "https://github.com/aws/aws-sam-cli/"
   url config_provider.url()
   sha256 config_provider.sha256
-  head "https://github.com/awslabs/aws-sam-cli.git", :branch => "develop"
+  head "https://github.com/aws/aws-sam-cli.git", :branch => "master"
```

Comment:
why move to master? being with branch develop allows for brew installs from source at the latest merged PR in develop.

Reply:
I think Appveyor is using this to build brew bottlers as part of release. Should it build against master instead of develop? Most times this shouldn't be a problem because the develop and master branches are even at the moment of build, but we need to avoid include some prs merge into develop at the last minute of build.

Reply:
I don't think this should be `master`. This key is used to get the latest source (which is develop). So customer can do `brew install sam --head` and get the lastest changes.

I would have to dig through the build bottle shell script (@c2tarun might know off hand) but we build bottles off a tag in the SAM repo not this `head`.

Reply:
Thanks for the backgrounds. I will close this PR and refine the build script instead.